api/src/main/java/uk/gov/dvsa/mot/app/ConfigKeys.java

```diff
@@ -12,10 +12,10 @@ public final class ConfigKeys {
     public static final String ProxyPort = "PROXY_PORT";
     public static final String LogLevel = "LOG_LEVEL";
 
-    public static final String SearchApiUrl = "SEARCH_API_URL";
-    public static final String SearchApiKey = "SEARCH_API_KEY";
-    public static final String SearchApiKeyEncrypted = "SEARCH_API_KEY_ENCRYPTED";
-    public static final String SearchApiConnectionTimeout = "SEARCH_API_CONNECTION_TIMEOUT";
+    public static final String MothApiUrl = "MOTH_API_URL";
+    public static final String MothApiKey = "MOTH_API_KEY";
+    public static final String MothApiKeyEncrypted = "MOTH_API_KEY_ENCRYPTED";
```

Comment:
My understanding is that we will no longer require an api key for the moth connection - however, we'll be in a better position to determine once the platform work has completed so content to mark this for later revision once the infrastructure is in place.